### DISCUSSION TITLE
Shift+P并点击高能进度条后的表现异常

### DISCUSSION THREAD

**Discussion by GFDGIT** [MuiseDestiny/zotero-style#397] [Q&A] [answered]
大佬您好，我遇到一个问题，不知道是设计如此还是bug，或者是设置的问题，麻烦您看一下，谢谢。

**步骤**：

1. 鼠标选中一个条目，按下Shift+P
2. 选择高能进度条
3. Shift+P面板会消失
4. 键盘焦点丢失，需要再次用鼠标选中条目
5. 按下Shift+P，能正常实现高能进度条

**预期行为**：
第2步选择高能进度条之后能直接显示，而不是需要再用鼠标选中条目并再按Shift+P。
已经尝试过disable其他插件，问题依然存在。

**其他建议**：

1. 在第5步正常显示进度条之后，如果点击其他条目，进度条会消失，需要再次按下Shift+P显示。 希望在鼠标点击/方向键选择其他条目的时候直接显示进度条。
2. 目前每次按ESC退出面板，就会丢失键盘焦点，需要在鼠标和键盘之间反复移动，是否可以通过右键Style图标来唤出面板？

**Comment by MuiseDestiny** [comment]
确实是，Bug了

**Reply by MuiseDestiny** [reply, marked as answer]
修复了

**Reply by GFDGIT** [reply]
这么神速的吗，感谢😁

**Reply by MuiseDestiny** [reply]
右键唤醒也加入了，失去焦点这个要后面修复。

**Reply by MuiseDestiny** [reply]
https://github.com/MuiseDestiny/zotero-style/releases/tag/2.6.5

安装下这个版本

**Reply by MuiseDestiny** [reply]
好的建议一般立马实施。感谢反馈。

**Reply by GFDGIT** [reply]
> https://github.com/MuiseDestiny/zotero-style/releases/tag/2.6.5
> 
> 安装下这个版本

安装了，一开始一切正常。然后我去测试标注预览，也是正常的，然后我多添加了几个标签，又删除了一些标签，中间应该是重启过zotero，不知怎么的，shift p高能进度条选项没了，标注预览也不出现了。暂时退回2.6.4，上面的问题还在，但是进度条选项也出现了。我明天换台电脑试一下。

**Reply by MuiseDestiny** [reply]
标注预览还在开发，还不是可用状态
